Nos/Views/Profile/ProfileView.swift

```diff
@@ -54,6 +54,24 @@ struct ProfileView: View {
         relaySubscriptions.append(await relayService.subscribeToEvents(matching: reportFilter)) 
     }
 
+    private var title: AttributedString {
+        let prefix = isShowingLoggedInUser ? String(
+            localized: LocalizedStringResource.localizable.yourProfile
+        ) : String(
+            localized: LocalizedStringResource.localizable.profileTitle
+        )
```

Comment:
Minor nitpick, but I'd find this easier to read if it were just three lines:

```suggestion
        let prefix = isShowingLoggedInUser ? 
            String(localized: LocalizedStringResource.localizable.yourProfile) :
            String(localized: LocalizedStringResource.localizable.profileTitle)
```